python-sdk/examples/dataset_kitchen_sink.py

```
@@ -0,0 +1,179 @@
+
+    print("=" * 60)
+    print("Dataset SDK Kitchen Sink Example")
+    print("=" * 60)
+
+    try:
+        # 1. Basic dataset retrieval
+        print("\n1. Basic dataset retrieval:")
+        print("-" * 60)
+        dataset = get_dataset(dataset_slug_or_id)
+        print(f"Retrieved dataset with {len(dataset.entries)} entries")
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)